frontend/src/components/PasswordReset/Reset.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+class Reset extends Component {
```

Comment:
Just to clarify, are we making a new page for password reset?

Reply:
Yes so the user will receive a url in their email, and that email will redirect them to a password reset page

Reply:
👍